### PR TITLE
PR for STACK-677

### PR DESCRIPTION
**Pull request by NehaKembalkarA10**
Fixture files are written here for the commands which are getting invoked for all the 4 facts classes used in acos_facts:

- Default

- Hardware

- Interfaces

- Config

The following fixture files contain the facts data generated by the respective commands on vThunder:

**acos_facts_show_bootimage** - show bootimage | include primary

**acos_facts_show_hardware** -  show hardware | include Serial, show hardware | include Thunder and show hardware | section Hardware 

**acos_facts_show_interfaces** - show interfaces 

**acos_facts_show_license** - show license 

**acos_facts_show_running_config** - show running-config 

**acos_facts_show_version** - show version 

**Note:**
These are untested fixtures which might get changed at the time of performing unit tests

### CODE REVIEW

File: test/units/modules/network/a10/fixtures/acos_facts_show_running_config [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+health monitor 4422d
+  retry 5
+  interval 3 timeout 2
+!
+health monitor udpudp2
+  override-ipv4 10.0.1.5
+  override-port 5555
+  method udp port 5555
+!
+health monitor omkar2
```
Comment (hthompson6) [inline test/units/modules/network/a10/fixtures/acos_facts_show_running_config]: Let's not have object names be dev names

File: test/units/modules/network/a10/fixtures/acos_facts_show_running_config [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+!
+slb server ok6 10.43.24.17
+!
+slb server ok7 10.43.24.18
+!
+slb server ok8 10.43.24.110
+!
+slb server ok9 10.43.24.111
+!
+slb server omkar 10.32.2.22
```
Comment (hthompson6) [inline test/units/modules/network/a10/fixtures/acos_facts_show_running_config]: server name shouldn't equal dev name

File: test/units/modules/network/a10/fixtures/acos_facts_show_running_config [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+!
+slb template http swapnilomAfrin-config4
+!
+slb template http adi-http-config1214
+!
+slb template http adi-http-config1112019
+!
+slb template http adi-http-config2813
+!
+slb template http swapnilomAfrin-config5
```
Comment (hthompson6) [inline test/units/modules/network/a10/fixtures/acos_facts_show_running_config]: Don't use dev names for test data

File: test/units/modules/network/a10/fixtures/acos_facts_show_running_config [hunk 4]
```diff
@@ -0,0 +1,257 @@
+!Current configuration: 1636 bytes
+!Configuration last updated at 10:46:27 IST Wed Jan 29 2020
```
Comment (hthompson6) [inline test/units/modules/network/a10/fixtures/acos_facts_show_running_config]: This should be test data should target specific info / edge cases. Seems to be a print out of a box we had laying around.

File: test/units/modules/network/a10/fixtures/acos_facts_show_running_config (hunk 1 above)
Comment (NehaKembalkarA10) [reply test/units/modules/network/a10/fixtures/acos_facts_show_running_config]: Deleted those objects which are having dev names to them

File: test/units/modules/network/a10/fixtures/acos_facts_show_running_config (hunk 2 above)
Comment (NehaKembalkarA10) [reply test/units/modules/network/a10/fixtures/acos_facts_show_running_config]: Changed the server names to the standard way like `server1`

File: test/units/modules/network/a10/fixtures/acos_facts_show_running_config (hunk 3 above)
Comment (NehaKembalkarA10) [reply test/units/modules/network/a10/fixtures/acos_facts_show_running_config]: Changed those dev names to standard names

File: test/units/modules/network/a10/fixtures/acos_facts_show_running_config (hunk 4 above)
Comment (NehaKembalkarA10) [reply test/units/modules/network/a10/fixtures/acos_facts_show_running_config]: done
